### ISSUE TITLE
VOC type mAP is only 0.25 and coco type is 0.11

### ISSUE THREAD

**Issue by Prakashvanapalli** [yhenon/pytorch-retinanet#34]
Hi,

Thanks for the repo. I am trying to calculate mAP on validation set and got the following results.

```
VOC type mAP is only 0.25 and coco type is 0.11
```

I have done the following things.
- Read an image
- Normalize using default mean and std
- Resize using padding
- network outputs
- nms and stuff
- rescale predicted bbox to original img dim.

I am not sure where I am making a mistake. Can u please help? Also which script are u using to calc mAP ?

Regards,
Prakash V
